source/includes/_samples.md

```diff
@@ -0,0 +1,66 @@
+# Sample Code
+
+## Resizing an `iframe` Based on Content Changes
+
+> iframe Code:
+
+```javascript
+document.body.style.overflow = 'hidden';
+
+function sendResizeMessage() {
+  window.parent.postMessage({
+    type: 'IFRAME_HEIGHT_RESIZE',
+    frameHeight: document.body.offsetHeight + 10 /* a little extra for good measure */
```

Comment:
Should we also require them to pass the ID of the iframe they're resizing here, in case they have multiple iframes on a page?

Reply:
I wasn't sure we wanted them generating ids that may not be unique. That's why I used a class instead. We're also not requiring them to write it just like this. I figured they could do whatever made sense for their situation. If we're expecting that they'll always have an id, we can document that if it's not already documented, and I'll make the update. Thoughts?

Reply:
I may have misspoke (mistyped?) -- I think what I meant to say is that the post message could take a 'target iframe', which could be a class name or ID or data attribute. That way you could have multiple iframes from the same domain on the page using the resize functionality and would be able to keep them separate in the parent page listener. The listener would just have to validate the domain name matches, then look for the target iframe and update its height. In this implementation you would only be able to maintain the height of one iframe per domain. This is not necessarily something we need to do in this documentation, as a developer could easily make that change since they handle both sides of the messaging and listening, but was just something that came to mind. :)

Reply:
I updated the code so the iframe sends along a target. This does indeed make the integration code more generic. Thanks! I also added a note about how the code may need to be modified to target iframes differently based on how the developer sets up their integration.